examples/CodeGen/deploy/gaudi.md

```diff
@@ -0,0 +1,375 @@
+# Single node on-prem deployment with TGI on Gaudi AI Accelerator
+
+This deployment section covers single-node on-prem deployment of the CodeGen
+example with OPEA comps to deploy using the TGI service. We will be showcasing how
+to build an e2e CodeGen solution with the CodeLlama-7b-hf model,
+deployed on Intel® Tiber™ AI Cloud (ITAC). To quickly learn about OPEA in just 5 minutes and set up the required hardware and software, please follow the instructions in the
```

Comment:
I'd make this a Hyperlink:
[Intel® Tiber™ AI Cloud (ITAC)](https://www.intel.com/content/www/us/en/developer/tools/tiber/ai-cloud.html)

Reply:
added